libraries/RW/K8s/k8s_connection.py

```diff
@@ -106,42 +110,34 @@ def shell(
         if (
             (rsp.status != 200 or rsp.returncode > 0)
             and rsp.stderr != ""
-            and not any(partial_stderr in rsp.stderr for partial_stderr in K8sConnectionMixin.ALLOWED_STDERR)
+            and not any(partial_stderr in rsp.stderr for partial_stderr in K8sConnection.ALLOWED_STDERR)
         ):
             raise ValueError(
                 f"The shell service responded with HTTP: {rsp.status} RC: {rsp.returncode} and response: {rsp}"
             )
         logger.info("shell stdout: %s", rsp.stdout)
         return rsp.stdout
 
+    @staticmethod
     def template_workload(
-        self,
         workload_name: str,
         workload_namespace: str,
-        workload_container: str,
-        target_service: platform.Service=None,
-        kubeconfig: platform.Secret=None,
-        context: str="",
+        workload_container: str
```

Comment:
same comment as below. This has a knock on effect to other codebundles at the moment. I think this was unintentional.